src/CoreShop/Component/Core/Order/Processor/CartSubtotalProcessor.php

```diff
@@ -33,8 +33,21 @@ public function process(OrderInterface $cart): void
          * @var OrderItemInterface $item
          */
         foreach ($cart->getItems() as $item) {
-            $subtotalGross += $item->getTotal(true);
-            $subtotalNet += $item->getTotal(false);
+            /**
```

Comment:
This is a comment and not a DocBlock, right?
```suggestion
            /*
```